src/app/hooks/AuthProvider.tsx

```diff
@@ -97,7 +96,7 @@ export function AuthProvider({ children }: { children: ReactNode }) {
 		setAuthLoading(false);
 	};
 
-	const permissions = TokenUtils.getPermissionsFromToken(egoJwt);
+	const permissions = TokenUtils.getPermissionsFromToken(egoJwt) || [];
```

Comment:
`getPermissionsFromToken` always returns `string[]` so this `|| []` is not needed.

https://github.com/icgc-argo/ego-token-utils/blob/develop/src/index.ts#L85